src/components/LogoutTile.jsx

```diff
@@ -14,10 +15,12 @@ const LogoutTile = () => {
     },
     [client]
   )
+  const { isMobile } = useBreakpoints()
```

Comment:
do we already have a breakpointprovider ?

Reply:
Yep!